### PR TITLE
Add ClaimedEth event on MVM claimEth calls

### LINKED ISSUE #253: Add an event for when the foundation claims ether from the MVM
We have an event for `sendTokens` but we are missing an event for `claimEth`

Thanks to CoinFabrik for reporting this issue as part of their audit

### PR DESCRIPTION
**Pull request by nberger**
Fixes #253

### CODE REVIEW

File: contracts/LifMarketValidationMechanism.sol [hunk 1]
```diff
@@ -62,6 +62,9 @@ contract LifMarketValidationMechanism is Ownable {
   event Pause();
   event Unpause(uint256 pausedSeconds);
 
+  event ClaimedEth(uint256 claimedWei);
```
Comment (AugustoL) [inline contracts/LifMarketValidationMechanism.sol]: it shouldnt be `claimedWei`? Its in wei units :)

Comment (nberger) [reply contracts/LifMarketValidationMechanism.sol]: I wanted to keep it consistent with the corresponding method name `claimEth` which also deals with wei amounts but it has eth in its name :)

Should we change the method name too? Maybe in a subsequent PR?

Comment (AugustoL) [reply contracts/LifMarketValidationMechanism.sol]: Yes I think we should, it claims it in Wei unit. So the next PR can be change Eth to Wei in naming.

Comment (nberger) [reply contracts/LifMarketValidationMechanism.sol]: Cool, I'll change both in a further PR, thanks!

**Pull request merged**